crane/container.go

```diff
@@ -69,15 +69,15 @@ type RunParameters struct {
 	RawCapAdd       []string    `json:"cap-add" yaml:"cap-add"`
 	RawCapDrop      []string    `json:"cap-drop" yaml:"cap-drop"`
 	RawCgroupParent string      `json:"cgroup-parent" yaml:"cgroup-parent"`
-	CpuPeriod       int         `json:"cpu-period" yaml:"cpu-period"`
```

Comment:
This is actually a rule from `golint`, taken from : https://github.com/golang/go/wiki/CodeReviewComments#initialisms
But I can remove them if you prefer :)